ldk/javascript/src/types/oliveHelps/whisper.d.ts

```diff
@@ -57,6 +57,24 @@ declare namespace WhisperService {
 
   type WidthSize = 'full' | 'half';
 
+  type CustomHeight =
```

Comment:
Are those enum enough for CustomHeight?

Reply:
> Are those enum enough for CustomHeight?

- Checked with Madison, CustomHeight will be integers ranging from 1rem to 36rem for now.

Reply:
> > Are those enum enough for CustomHeight?
> 
> * Checked with Madison, CustomHeight will be integers ranging from 1rem to 36rem for now.

Updated customHeight to T-shirt sizes accordingly